### PR TITLE
Verify Noobaa UI on OCP4.7

### PR DESCRIPTION
**Pull request by OdedViner**
Signed-off-by: Oded Viner <oviner@redhat.com>

### CODE REVIEW

File: ocs_ci/ocs/ui/deployment_ui.py [hunk 1]
```diff
@@ -279,6 +279,26 @@ def verify_operator_succeeded(
...
 
+    def verify_noobaa_ui(self):
+        """
+        Verify Noobaa UI
+
+        """
+        self.navigate_overview_page()
+        self.do_click(self.dep_loc["object_service_tab"])
+        platform = config.ENV_DATA.get("platform").lower()
+        if platform == constants.VSPHERE_PLATFORM:
```
Comment (ramkiperiy) [inline ocs_ci/ocs/ui/deployment_ui.py:290]: any plans to add verification for AWS ? or we have it covered somewhere else

Comment (OdedViner) [reply ocs_ci/ocs/ui/deployment_ui.py:290]: On AWS, I  search "strings" on relevant html

File: ocs_ci/ocs/resources/storage_cluster.py [hunk 2]
```diff
@@ -399,6 +400,23 @@ def ocs_install_verification(
             f"The failure domain type on LSO cluster with {zone_num} zones should be "
             f"'host' and not {failure_domain}."
         )
+    verify_noobaa_console()
+
+
+def verify_noobaa_console():
```
Comment (ebenahar) [inline ocs_ci/ocs/resources/storage_cluster.py:406]: I suggest we create a class that verifies a page and other child classes that verify specific pages, such as noobaa console

**Comment by MeridianExplorer** [comment]
@OdedViner - Why was this closed?

**Comment by OdedViner** [comment]
> @OdedViner - Why was this closed?

@MeridianExplorer I am working on tier1 ui test, which checks all the relevant pages (not only noobaa page)

**Pull request closed without merging**